airqo-platform/AirQo-frontend
# Ft location device registry

What does this PR do?
    This PR created a UI for the location registry component of Airqo Netmanager.
    What JIRA task is related to this PR?
    
    How do I test out this PR?
    Git clone this repo cd AirQo-frontend/locate Then run npm install to get all the necessary dependencies Afterwards run npm run local. Make sure you're running the location-registry service from the [location-registry-backend](https://github.com/airqo-platform/AirQo-api/tree/location-registry-backend) branch
    Which end points should are ready for testing?
    None
